pylint/testutils/_primer/primer_run_command.py

```diff
@@ -90,21 +80,12 @@ def _lint_package(
             pylint_exit_code = int(e.code)
         readable_messages: str = output.getvalue()
         messages: list[OldJsonExport] = json.loads(readable_messages)
-        astroid_errors: list[Message] = []
-        other_fatal_msgs: list[Message] = []
+        fatal_msgs: list[Message] = []
         if pylint_exit_code % 2 == 0:
             print(f"Successfully primed {package_name}.")
         else:
-            astroid_errors, other_fatal_msgs = self._filter_astroid_errors(messages)
-            print(f"Encountered fatal errors while priming {package_name} !\n")
-            if other_fatal_msgs:
-                print(
-                    "Fatal messages unrelated to astroid:\n"
-                    f"{self._print_msgs(other_fatal_msgs)}\n\n"
-                )
-            if astroid_errors:
-                warnings.warn(
-                    f"Fatal messages that could be related to bleeding edge astroid:\n"
-                    f"{self._print_msgs(astroid_errors)}\n\n"
-                )
-        return messages, astroid_errors, other_fatal_msgs
+            fatal_msgs = self._filter_fatal_errors(messages)
+            if fatal_msgs:
+                print(f"Encountered fatal errors while priming {package_name} !\n")
+                print(f"{self._print_msgs(fatal_msgs)}\n\n")
```

Comment:
If we `print` instead of `warnings.warn` then it doesn't summarize as nicely in Github Actions (it will get lost).